extensions/interactions/ImageClickInput/directives/ImageClickInput.js

```diff
@@ -47,7 +48,32 @@ oppia.directive('oppiaInteractiveImageClickInput', [
               '/' + encodeURIComponent($scope.filepath)) : null);
           $scope.mouseX = 0;
           $scope.mouseY = 0;
-          $scope.currentlyHoveredRegions = [];
+          $scope.deactivate = false;
+          if ($scope.currentCard.destStateName) {
+            $scope.deactivate = true;
+            $scope.currentlyHoveredRegions = [];
+            /*Following lines to permanently highlight the learner's previous
+              answer. */
+            $scope.mouseX = $scope.currentCard.inputResponsePairs[
+                              $scope.currentCard.inputResponsePairs.length - 1]
+                                .learnerInput.clickPosition[0];
+            $scope.mouseY = $scope.currentCard.inputResponsePairs[
+                              $scope.currentCard.inputResponsePairs.length - 1]
+                                .learnerInput.clickPosition[1];
+            for (var i = 0; i < imageAndRegions.labeledRegions.length; i++) {
+              var labeledRegion = imageAndRegions.labeledRegions[i];
+              var regionArea = labeledRegion.region.area;
+              if (regionArea[0][0] <= $scope.mouseX &&
+                  $scope.mouseX <= regionArea[1][0] &&
+                  regionArea[0][1] <= $scope.mouseY &&
+                  $scope.mouseY <= regionArea[1][1]) {
+                $scope.currentlyHoveredRegions.push(labeledRegion.label);
+              }
+            }
+          }
+          else {
```

Comment:
Lint nit: move to end of previous line.

Reply:
I removed this else as it was actually not needed.